src/AsyncGenerator.yml

```diff
@@ -183,6 +183,18 @@
   applyChanges: true
   analyzation:
     methodConversion:
+    - conversion: Ignore
+      name: CanUseDependentTransaction
+      containingTypeName: DistributedSystemTransactionFixture
+    - conversion: Ignore
+      name: CanUseSessionWithManyDependentTransaction
+      containingTypeName: DistributedSystemTransactionFixture
+    - conversion: Ignore
+      name: CanUseDependentTransaction
+      containingTypeName: SystemTransactionFixture
+    - conversion: Ignore
+      name: CanUseSessionWithManyDependentTransaction
+      containingTypeName: SystemTransactionFixture
```

Comment:
The issue was indeed since 0.14.0 when the `NUnitPlugin` was introduced. `NUnitPlugin` rules currently have a higher priority that the rules defined by the user which is something that I overlooked. By just swaping the priorities will not work for NHibernate, because of the following rule:
```
- conversion: Ignore
  anyBaseTypeRule: IsTestCase
```
which is supposed to run last (after `NUnitPlugin` rules), otherwise most of the tests would not be generated. In order to support that I've made maca88/AsyncGenerator/pull/138, which allows to define when the rule should be executed by using a new option `executionPhase`:
```
- conversion: Ignore
  anyBaseTypeRule: IsTestCase
  executionPhase: PostProviders
``` 
Should I create a new PR for fixing this issue or update #2182?

Reply:
Ok, thanks for looking into this. Updating #2182 is fine I think.